src/main/kotlin/com/hibob/academy/resource/OwnerApi.kt

```diff
@@ -0,0 +1,86 @@
+package com.hibob.academy.resource
+
+import com.hibob.restAPI.Owner
+import jakarta.ws.rs.*
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+import java.util.concurrent.CopyOnWriteArrayList
+
+@Controller
+@Path("/api/adi/owners")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class OwnerResource {
+
+    private val allOwners: MutableList<Owner> = CopyOnWriteArrayList()
+
+    @GET
+    @Path("/{ownerId}")
+    fun getOwnerById(@PathParam("ownerId") ownerId: Long): Response {
+        val owner = allOwners.find { it.id == ownerId.toInt() }
```

Comment:
The kotlin way would be to use `owner?.let { } ?: ` instead of the if else and null check.